Makefile

```diff
@@ -233,9 +244,11 @@ lua_modules/bin/rover:
 dependencies: dep_folders lua_modules carton  ## Install project dependencies
 
 clean-containers:
-	- $(DOCKER_COMPOSE) down --volumes --remove-orphans
-	- $(DOCKER_COMPOSE) -f $(PROVE_DOCKER_COMPOSE_FILE) down --volumes --remove-orphans
-	- $(DOCKER_COMPOSE) -f $(DEVEL_DOCKER_COMPOSE_FILE) -f $(DEVEL_DOCKER_COMPOSE_VOLMOUNT_FILE) down --volumes --remove-orphans
+	$(DOCKER) compose down --volumes --remove-orphans
```

Comment:
I think this removes any potential compatibility with `podman-compose` (not sure if it was working beforehand), as it is a separate command and not a subcommand for `podman`.

Reply:
In this repo, we use `docker`, not podman. I do not think it works with podman, even before this change. Check this [PR](https://github.com/3scale/APIcast/pull/1353) for a discussion regarding Podman vs Docker. I would be happy to move to podman/buildah. But it needs work and maybe it cannot be done.

This is about moving forward to docker compose V2. Docker compose V1 is deprecated  https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/